src/domain/behaviours/BacktrackingBehaviour/BasicBacktrackingBehaviour.ts

```diff
@@ -108,16 +108,22 @@ export class BasicBacktrackingBehaviour implements IBackTrackingBehaviour {
       findBlockOnActiveFlowWith(intx.blockId, context),
       context)
   }
-
-  async peek(steps = 0, context: IContext = this.context): Promise<IRichCursorInputRequired> {
+  
+  _findInteractiveInteractionAt(steps = 0, context: IContext = this.context, fromLeft = false): IBlockInteraction {
```

Comment:
the fromLeft is a boolean parameter. Would it make sense to use a enum instead for call site clarity?

Reply:
Ya, I love this suggestion <3